library/bignum.c

```diff
@@ -1330,21 +1330,39 @@ int mbedtls_mpi_add_abs( mbedtls_mpi *X, const mbedtls_mpi *A, const mbedtls_mpi
 /*
  * Helper for mbedtls_mpi subtraction
  */
-static void mpi_sub_hlp( size_t n, mbedtls_mpi_uint *s, mbedtls_mpi_uint *d )
+static void mpi_sub_hlp( const mbedtls_mpi *X, const mbedtls_mpi *A, const mbedtls_mpi *B )
```

Comment:
Suggestion for improvement: I think it would be worth updating the comment above the function while we're at it, to mention that the function assumes `X->n >= A->n` and it's the caller's job to make sure this holds. (It's not worth checking it in the code as it's a static function, but it still should be documented.)

Note: this is a pre-existing issue, a lot of our internal functions are under-documented, so it's not a problem with your PR, but since we're touching that function we might as well improve its documentation too.

Reply:
Makes sense.  I updated the comment.